Configuration files for New Zealand

maciejtarsa opened the pull request:
This pull requests contains configuration files for New Zealand. All 16 regions are included. In the demographic file, these are divided into 2133 statistical areas with accurate population totals, genders, ethnicities, age ranges, income and education levels, all obtained from the 2018 national census. The provider files only contains public hospitals (84 of them) obtained from the government website, these are copied over to primary care facilities and urgent care facilities files.

maciejtarsa committed:
Add files via upload

File for New Zealand

maciejtarsa committed:
Create readme.md

Details about the source of data provided

jawalonoski commented:
There are still a few issues with patients being unable to find certain types of care, but I debugged the issue, and it looks like those patients are military veterans, who are seeking to go to Veteran's hospitals and can't find any.

If NZ does not have Veteran-specific hospitals, you can just copy the contents of `hospitals.csv` onto `va_facilities.csv` and change the contents of the `type` column to be exactly equal to `VA Facility` instead of `Public Hospital` (or whatever value is there). Apologies that this is not documented well.

maciejtarsa committed:
Add files via upload

'Area Outside' removed

maciejtarsa committed:
Add files via upload

'Area Outside' removed; file 'va_facilities.csv' added

maciejtarsa commented:
Thanks for spotting this. I have now added the file. I cannot find any information about specific facilities for veterans in NZ, so I copied the file as you suggested.

I also removed any reference from the files to 'Area Outside'. There was one area which was outside of any region in geography files and I had one medical centre assigned to that area, but all patients were coming up with 'cannot find provider'. As it is just one location, I decided to remove it.

Other than that, it seems to be working now for all regions.

jawalonoski commented:
Thank you for the pull request and the contribution! I tested it locally and found no issues.
